src/test/java/test/configuration/github1625/TestclassSampleUsingMocks.java

```diff
@@ -0,0 +1,30 @@
+package test.configuration.github1625;
+
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class TestclassSampleUsingMocks {
+    @Mock
+    List<String> list;
+
+
+    @BeforeClass
+    public void beforeClass() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
Fyi, a `MockitoTestNGListener` exists too (but it won't follow the sample from the issue) :)